apps/learner/src/routes/(app)/+page.svelte

```diff
@@ -1,19 +1,30 @@
 <script lang="ts">
+  import { ArrowLeft } from '@lucide/svelte';
+
   import FloatingChat from '$lib/components/FloatingChat.svelte';
   import FloatingPlayer from '$lib/components/FloatingPlayer.svelte';
   import LearningUnit from '$lib/components/LearningUnit.svelte';
 
   let isFloatingPlayerVisible = $state(false);
   let isPlaying = $state(false);
+  let isModalOpen = $state(false);
 
-  function handlePlay() {
+  const handlePlay = () => {
     isFloatingPlayerVisible = true;
     isPlaying = true;
-  }
+  };
 
-  function togglePlayPause() {
+  const togglePlayPause = () => {
     isPlaying = !isPlaying;
-  }
+  };
+
+  const openModal = () => {
```

Comment:
Maybe can rename to `handleFloatingPlayerClick` or something since it would only execute when the floating player is clicked.